File: Kap11/C/Listing15/SwitchTest.c
#include <stdio.h>

void printWeekDay(int day) {
  switch(day) {
    case 1:
      printf("Montag");
      break;
    case 2:
      printf("Dienstag");
      break;
    case 3:
      printf("Mittwoch");
      break;
    case 4:
      printf("Donnerstag");
      break;
    case 5:
      printf("Freitag");
      break;
    default:
      printf("Wochenende");
  }
}

int main() {
    printWeekDay(2);
    return 0;
}

File: Kap11/Python/Listing16/SwitchTest.py
def printWeekday(day):
  match day:
    case (1):
      print("Montag")
    case (2):
      print("Dienstag")
    case (3):
      print("Mittwoch")
    case (4):
      print("Donnerstag")
    case (5):
      print("Freitag")
    case _:
      print("Wochenende")


printWeekday(2)